src/ontology/ro-edit.owl

```diff
@@ -1224,18 +1225,42 @@ AnnotationAssertion(rdfs:label subsets:ro-eco "eco subset")
 AnnotationAssertion(rdfs:seeAlso subsets:ro-eco <http://purl.obolibrary.org/obo/ro/docs/subsets/>)
 SubAnnotationPropertyOf(subsets:ro-eco oboInOwl:SubsetProperty)
 
+# Annotation Property: dc:creator (creator)
+
+AnnotationAssertion(rdfs:label dc:creator "creator")
+
+# Annotation Property: dc:date (date)
+
+AnnotationAssertion(rdfs:label dc:date "date")
+
 # Annotation Property: oboInOwl:SubsetProperty (subset_property)
 
 AnnotationAssertion(rdfs:label oboInOwl:SubsetProperty "subset_property")
 
-# Annotation Property: oboInOwl:hasNarrowSynonym (has_narrow_synonym)
+# Annotation Property: oboInOwl:hasNarrowSynonym (has narrow synonym)
 
-AnnotationAssertion(rdfs:label oboInOwl:hasNarrowSynonym "has_narrow_synonym")
+AnnotationAssertion(rdfs:label oboInOwl:hasNarrowSynonym "has narrow synonym")
 
 # Annotation Property: oboInOwl:inSubset (in_subset)
 
 AnnotationAssertion(rdfs:label oboInOwl:inSubset "in_subset")
 
+# Annotation Property: rdfs:label (label)
+
+AnnotationAssertion(rdfs:label rdfs:label "label")
+
+# Annotation Property: rdfs:seeAlso (see also)
+
+AnnotationAssertion(rdfs:label rdfs:seeAlso "see also")
+
+# Annotation Property: skos:closeMatch (close match)
+
+AnnotationAssertion(rdfs:label skos:closeMatch "close match")
+
+# Annotation Property: skos:exactMatch (exact match)
+
+AnnotationAssertion(rdfs:label skos:exactMatch "exact match")
```

Comment:
This information (labels etc) must be imported from OMO rather than hard coded in the edit file!

Reply:
Just remove all these AnnotationAssertions again and the PR is fine.

Reply:
done